homework3/semi-uber/Server/api/handlers/loadsHandlers/loadsPostHandler.js

```diff
@@ -0,0 +1,38 @@
+const { Load, loadSchemaValidation } = require("../../Schemas/load.schema");
+const log4js = require("log4js");
+const logger = log4js.getLogger();
+const errorHandler = require("../error.handler");
+
+const loadsPostHandler = (req, res) => {
+  const { id, dimensions, payload } = req.body;
+  const load = {
+    created_by: id,
+    logs: [{ message: "Created load", time: new Date().getTime() }],
+    assigned_to: "Not assigned",
```

Comment:
It will be more convenient to set `null` here if it is not assigned, then it will be really ease to check if the load is assigned to someone or not